projects/templates/src/lib/components/po-page-dynamic-table/po-page-dynamic-table.component.ts

```diff
@@ -400,8 +401,28 @@ export class PoPageDynamicTableComponent extends PoPageDynamicListBaseComponent
     this.navigateTo({ path, url });
   }
 
-  private openNew(path: string) {
-    this.navigateTo({ path });
+  private openNew(actionNew: PoPageDynamicTableActions['new']) {
+    this.subscriptions.add(
+      this.poPageDynamicTableActionsService
+        .beforeNew(this.actions.beforeNew)
+        .subscribe((beforeNewResult: PoPageDynamicTableBeforeNew) =>
+          this.executeNew(actionNew, beforeNewResult.allowAction, beforeNewResult.newUrl)
+        )
+    );
+  }
+
+  private executeNew(actionNew: PoPageDynamicTableActions['new'], allowAction = true, newUrl?: string) {
```

Comment:
Ao retornar um allowAction com nullo, o mesmo não esta deixando chamar a nova URL.

Deve bloquiar apenas quando for "false".

Reply:
Show muito obrigado.
Também criei um testes para garantir esse comportamento